include/wx/helpwin.h

```diff
@@ -1,19 +0,0 @@
-/////////////////////////////////////////////////////////////////////////////
```

Comment:
We shouldn't delete this file, even if it's useless now, it's quite possible that existing code includes it and not `wx/msw/helpwin.h` and we shouldn't break it.